templates/default/fulldoc/html/css/style.css

```diff
@@ -106,6 +112,7 @@ h2 small a {
   position: relative;
   padding: 2px 7px;
 }
+a { font-weight: 550; }
```

Comment:
Bold links are 700, normal is 400. This is a weight in between. The requirement was to make links more obvious for people with low vision.